tests/wrappers/test_multioutput.py

```diff
@@ -0,0 +1,29 @@
+import torch
+
+from torchmetrics.classification import Accuracy
+from torchmetrics.regression import R2Score
+from torchmetrics.wrappers.multioutput import MultioutputWrapper
+
+
+def test_multioutput_wrapper():
```

Comment:
For testing it would be great if we tested on random input instead of specific

Reply:
Done!